go-selinux/selinux_linux.go

```diff
@@ -583,7 +583,8 @@ func bitsetToStr(c *big.Int) string {
 	var str string
 
 	length := 0
-	for i := int(c.TrailingZeroBits()); i < c.BitLen(); i++ {
+	i0 := int(c.TrailingZeroBits()) //nolint:gosec
```

Comment:
Same here, and possible add a comment after the `// #nosec G115 -- <short commend here >`

Reply:
Done